kuma/core/helpers.py

```diff
@@ -150,8 +151,11 @@ def entity_decode(str):
 
 
 @register.function
-def inlinei18n(locale):
-    return StaticI18nJob().get(locale)
+def statici18n(locale):
+    if not locale:
+        locale = settings.LANGUAGE_CODE
+    filename = get_filename(locale, settings.STATICI18N_DOMAIN)
+    return static(os.path.join(settings.STATICI18N_OUTPUT_DIR, filename))
```

Comment:
I was thinking *build/locale* so that the files would be placed like *build/locale/es/javascript.js*. The build directory is being created and added to `STATICFILES_DIRS` as part of #3614, so by putting the *locale* directory in there `collectstatic` will see it and copy it to `/static/locale` with a versioned filename.